styles.css

```diff
@@ -310,3 +310,73 @@ ul.technologies_container-languages > li > p {
 .border_bottom {
   border-bottom: 2px solid #dfe1e6;
 }
+
+/* contact form */
+article.contact_me {
+  background-image: url(images/contact_form_shapes-mobile@2x.png);
+  background-size: 98% 92%;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: auto;
+  margin-top: 3rem;
+  border-top-left-radius: var(--radiusL);
+  background-color: var(--primaryb400);
+  padding: 5rem 20px 20px 20px;
+}
+
```

Comment:
- [ ] Kindly position the `background-image` for this section properly. The background-image in your design for this section looks slanty and very big. I would advise you to use the correct background-image and position them properly. 👍 😎 

### Your Design

![image](https://github.com/MPDADDY/my_portfolio/assets/77106962/6e5a1a13-30ed-4848-993a-d1c6aba44794)

### Figma Design

![image](https://github.com/MPDADDY/my_portfolio/assets/77106962/1e65eea3-e509-4811-a2fa-ad54225a9a29)